server/checks/Approval.js

```diff
@@ -118,39 +118,38 @@ export default class Approval extends Check {
   }
 
   /**
-   * Returns the comment if it matches the config, null otherwise.
+   * Returns `true` if username is in the config, false `otherwise`.
```

Comment:
Just for consistency, `comment` wasn't used, but return value was checked inside conditions. Seems like booleans are more suitable for that.